app/javascript/flavours/glitch/utils/dom_helpers.js

```diff
@@ -1,10 +1,6 @@
 //  Package imports.
 import { supportsPassiveEvents } from 'detect-passive-events';
```

Comment:
This import should also be deleted, as it is unused now.

Reply:
Oops! Fixing that!